services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket],
+                 username: str,
+                 date: str = None) -> QuerySet:
```

Comment:
The `date` argument is optional, but there is no default value provided. This can lead to a `TypeError` if the `date` argument is not provided when the function is called. You should provide a default value for `date`.